### PR TITLE
Add hint to Private SSH Key on Cluster Import

### PR DESCRIPTION
**Pull request by kellystrauch0**
* adds hint describing which Private SSH Key user should input
* resolves card S20-1047

## Type

- [ ] Fix (a **bug** or other **issue** was fixed)
- [ ] Feature (**new functionality** was added)
- [ ] Breaking (**existing functionality** was affected)
- [x] Other (please **explain** below)
UI change only

## Details

Example: Introduces a new cloud provider for provisioning and managing kubes on GCE, due to popular demand.

Example: Fixes #78, #89, #123

### Additional Details?

Example: In order to take advantage of this new feature, users will need to restart SG Control with the `gce` provider option added to the `providers` flag: `--providers=gce,aws,etc.` Otherwise, functionality is not changed.

## Checklist

- [ ] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **needed**, and I updated it.
- [x] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **not** needed.
- [x] I understand [the Contribution Guidelines](https://supergiant.readme.io/docs/guidelines).

### CODE REVIEW

File: cmd/ui/assets/src/app/clusters/import-cluster/import-cluster.component.html [hunk 1]
```diff
@@ -90,6 +90,7 @@ <h1>Import Kubernetes Cluster</h1>
           </mat-form-field>
           <mat-form-field>
             <textarea matInput rows="4" formControlName="privateKey" placeholder="Private SSH Key" required></textarea>
```
Comment (stgleb) [inline cmd/ui/assets/src/app/clusters/import-cluster/import-cluster.component.html]: I think we need to point out that this key must allow ssh-ing with ubuntu user.

Comment (stgleb) [inline cmd/ui/assets/src/app/clusters/import-cluster/import-cluster.component.html]: I think we need to point out that this key must allow ssh-ing with ubuntu user.

**Comment by stgleb** [comment]
I would say - "Please, provide ssh key that allows ssh to cluster machines with ubuntu user. If you are importing a Supergiant-created cluster, you can download the needed Private Key from the cluster's detail page. "

**Comment by eldridgeh** [comment]
@kellystrauch0 
```
Add the Private SSH Key of the Ubuntu user for the Kubernetes cluster you wish to import. If you are importing a Supergiant-created cluster, you can download the needed Private Key from the cluster's detail page.
```

**Pull request merged**